acm_certificate/main.tf

```diff
@@ -50,6 +50,13 @@ resource "aws_acm_certificate" "main" {
   lifecycle {
     create_before_destroy = true
   }
+
+  # TODO: this is a workaround for an AWS API / Terraform AWS provider bug
+  # https://github.com/terraform-providers/terraform-provider-aws/issues/8531
+  # https://github.com/18F/identity-devops/issues/1469
+  lifecycle {
+    ignore_changes = ["subject_alternative_names"]
```

Comment:
Unless I'm downloading from the ref incorrectly, I think this and the previous `lifecycle` block need to be combined:
```
Error downloading modules: Error loading modules: module acm-cert-apps-combined: Error loading .terraform/modules/f13ed9500691ee81a9c4c1bf84d107f2/acm_certificate/main.tf: aws_acm_certificate[main]: Multiple lifecycle blocks found, expected one
```

Reply:
Done